ui/v2.5/src/docs/en/Manual/Plugins.md

```diff
@@ -10,7 +10,43 @@ Plugin tasks can be implemented using embedded Javascript, or by calling an exte
 
 > **⚠️ Note:** Plugin support is still experimental and is likely to change.
 
-# Adding plugins
+## Managing Plugins
+
+Plugins can be installed and managed from the `Settings > Plugins` page. 
+
+Scrapers are installed using the `Available Plugins` section. This section allows configuring sources from which to install plugins. The `Community (stable)` source is configured by default. This source contains plugins for the current _stable_ version of stash.
+
+These are the plugin sources maintained by the stashapp organisation:
+
+| Name | Index URL | Recommended Local Path | Notes |
```

Comment:
Just to be consistent, "Index URL" should probably rather be "Source URL".

Reply:
Hmm okay well I've checked the actual UI, and it uses "Index URL" - so this is fine, but the other mentions of "Source URL" should be "Index URL" instead.

Although I think "Source URL" is a more user-friendly term than "Index URL", so maybe we should change the UI to "Source URL" or just "URL".

Reply:
I also like `Source URL` more.